applications/audio/broadcast_config_tool/src/main.c

```diff
@@ -674,6 +811,45 @@ static int argv_to_indexes(const struct shell *shell, size_t argc, char **argv,
 	return 0;
 }
 
+static int enable_big(const struct shell *shell, uint8_t big_index)
+{
+	int ret;
+
+	if (big_index >= CONFIG_BT_ISO_MAX_BIG) {
+		return -EINVAL;
+	}
+
+	if ((broadcast_param[big_index].subgroups == NULL) ||
+	    (broadcast_param[big_index].num_subgroups == 0)) {
+		LOG_ERR("No subgroups defined for BIG%d", big_index);
+		return -EINVAL;
+	}
+
+	ret = broadcast_source_enable(&broadcast_param[big_index], big_index);
+	if (ret) {
+		shell_error(shell, "Failed to enable broadcaster: %d", ret);
+		return ret;
+	}
+
+	ret = adv_create_and_start(shell, big_index);
+	if (ret) {
+		shell_error(shell, "Failed to start advertising for BIG%d: %d", big_index, ret);
+		return ret;
+	}
+
+	/* TODO: Workaround this. Must find a new way. */
+	k_msleep(100);
```

Comment:
Could you use the LE_AUDIO_EVT_STREAMING evt to kick off the streaming?

Reply:
Done, seems to work nice!